spec/helpers/jwt_auth_helpers_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe JWTAuthHelpers, type: :helper do
+  describe '#user_info_from_header' do
+    it 'should return nil when the request does not have an Authorization header' do
+      allow(helper).to receive(:headers).and_return({})
+      expect(helper.send(:user_info_from_header)).to eq(nil)
```

Comment:
Nah I'd say we're good because the intent here was different than a normal private. We don't want to allow access but it looked better in the method IMO